containerization-installs/docker-install.sh

```diff
@@ -28,7 +28,7 @@ fi
 # Creating a keyrings directory 
 if (test -d /etc/bin/keyrings)
```

Comment:
Paths in the if(test...) don't match what gets created in the else. Ik this wasn't a change in this pr, but was this done on purpose?  
Same with ln40

Reply:
A lot of this was just slapped together months ago. So it's is very likely not right, thank you Greg for the input👍